scripts/run_benchmark.sh

```diff
@@ -36,5 +36,5 @@ $BENCHMARK \
   --steps 50 \
   --repeat 10 \
   --output=$weights_file \
-  --template=./.maintain/runtime-weight-template.hbs
+  --template=./.maintain/frame-weight-template.hbs
```

Comment:
any reason why this is changed?

Reply:
The runtime benchmark template doesn't write a `WeightInfo` trait definition to the benchmark file.